ui/theme/theme.go

```diff
@@ -55,19 +57,27 @@ func NewMyTheme(config *backend.ThemeConfig, themeFileDir string) *MyTheme {
 	m := &MyTheme{config: config, themeFileDir: themeFileDir}
 	m.defaultThemeFile, _ = DecodeThemeFile(bytes.NewReader(res.ResDefaultToml.StaticContent))
 	m.createThemeIcons()
+	if err := m.createThemeResources(); err != nil {
+		log.Printf("failed to create theme resources: %s", err.Error())
+	}
 	return m
 }
 
 func (m *MyTheme) Color(name fyne.ThemeColorName, _ fyne.ThemeVariant) color.Color {
 	// load theme file if necessary
 	if m.loadedThemeFile == nil || m.config.ThemeFile != m.loadedThemeFilename {
+		if m.config.ThemeFile == "" {
```

Comment:
These lines in particular, were critical to add, since without them, if the `ThemeFile` was empty, the application would try to join the themes path with an empty string, leading Supersonic to try to read the themes directory as a toml configuration file.

This, aside from the stated reasons, is why I decided to dump the default configuration into a file.